### ISSUE TITLE
 Provide a way to automate backups

### ISSUE THREAD

**Issue by vyaslav** [vertica/vertica-kubernetes#209]
Currently to automate backups we've implemented UDx in python to run vbr and trigger it via a simple cronjob:
```yaml
apiVersion: batch/v1beta1
kind: CronJob
metadata:
  name: vertica-backup
spec:
  schedule: "0 * * * *"
  jobTemplate:
    spec:
      template:
        spec:
          restartPolicy: OnFailure
          containers:
            - name: vertica-backup
            ...
```
But this doesn't work for multi-node setups and it would be great to have an official way of doing the same.

**Comment by spilchen** [comment]
It is part of our road map to support backup/restore through the operator.  However, the server team is in the process of migrating vbr functionality into the server.  This will create a new backup/restore interface.  So, we are holding off at adding a vbr wrapper in the operator.

**Comment by spilchen** [comment]
Backup/restore integration is next on our roadmap. We are currently finishing up the removal of SSH. We hope to start the integration in January. However, it is important to note that the integration will not use the vbr script. The server team is developing a new backup architecture that is integrated directly into the server. We will be integrating that into the operator.

While we work on improving integration, we have published some tips on using vbr in k8s here to help bridge the gap.: https://docs.vertica.com/23.3.x/en/containerized/backup-restore/

**Comment by spilchen** [comment]
We've just released version 24.2.0 of the Vertica server, introducing beta support for in-database backup and restore using SQL. The latest version of the operator now accommodates this new functionality. For details, refer to the Vertica documentation on VerticaDB CRD parameters: https://docs.vertica.com/24.2.x/en/new-features/24.2/containers-kubernetes/#verticadb-crd-parameters.

Originally, this issue was raised to devise an automated method for scheduled backups. We've now documented this process using a stored procedure within the server: https://docs.vertica.com/24.2.x/en/containerized/custom-resource-definitions/vertica-restorepoints-query/#save-restore-points.
